docs/README.md

```diff
@@ -0,0 +1,19 @@
+# Welcome to Win32 App Isolation Introduction
+
+### Target Applications
+
+* Win32
+* Centennial apps.
+
+### Minimum requirements:
+
+* Windows insider version >= 25357
+* [MPT v1.2023.517.0] (https://github.com/microsoft/win32-app-isolation/releases/tag/v0.1.1)
```

Comment:
Remove the space between `]` and `(`. The correct grammar is `(link string)[actual link]`.

The version is actually not important - because the official MPT does not work even after this version number. This is a customized version of MPT, so just say `Customized MSIX Packaging Tool`. Then add a description after saying you have to use our version.